docs/content/modeling/migrating/migrating-schema-1-1.mdx

```diff
@@ -490,6 +490,15 @@ It is possible to introduce new models and have existing tuples (from prior mode
 
 In these cases, <ProductName format={ProductNameFormat.ShortForm}/> will not consider those invalid tuples when evaluating queries ([Check](/api/service#/Relationship%20Queries/Check), [Expand](/api/service#/Relationship%20Queries/Expand), [List-objects](/api/service#/Relationship%20Queries/ListObjects), etc). However, after any of the changes above happen, you should delete those tuples as having a large number of invalid tuples will negatively affect performance.
 
+## Improved Schema Validation
+
+Type restriction allow <ProductName format={ProductNameFormat.ShortForm} /> to validate schema better at the time of writing the schema instead of at time of query evaluation.
```

Comment:
```suggestion
Type restrictions allow <ProductName format={ProductNameFormat.ShortForm} /> to validate the schema better at the time of writing the schema instead of at the time of query evaluation.
```